calico-enterprise/networking/egress/egress-gateway-aws.mdx

```diff
@@ -1175,6 +1176,45 @@ patch release.
 
 :::
 
+## Prefer local egress gateways
+
+From v3.18, preference can be set to use the local egress gateways. `GatewayPreference` can be set to `PreferNodeLocal` or `None` in the Egress Gateway Policy. By default `GatewayPreference` is set to `None`.
+When set to `PreferNodeLocal`, egress gateways local to the client's node will be used if available. If there are no local egress gateways, we will fallback to using the other egress gateways.
+
+The following is an example of Egress Gateway Policy with GatewayPreference configured.
+
+```yaml noValidation
+apiVersion: projectcalico.org/v3
+kind: EgressGatewayPolicy
+metadata:
+  name: "egw-policy1"
+spec:
+  rules:
+  // callout-for-next-line
```

Comment:
What does this special comment do (my guess is that it makes a 1, 2, 3 appear int eh example so you can reference it below but I'm not sure you're doing that)?

Reply:
Yes right. This is not needed.